azure-mgmt-eventhub/src/test/java/com/microsoft/azure/management/eventhub/EventHubTests.java

```diff
@@ -166,18 +166,45 @@ public void canManageEventHubNamespaceEventHubs() throws Exception {
     }
 
     @Test
-    public void canManageEventHubNamespaceAuthorizationRules() throws Exception {
+    public void canManageEventHubNamespaceWithKafka() throws Exception {
         RG_NAME = generateRandomResourceName("javacsmrg", 15);
         final String namespaceName = SdkContext.randomResourceName("ns", 14);
 
         EventHubNamespace namespace = eventHubManager.namespaces()
                 .define(namespaceName)
                     .withRegion(REGION)
                     .withNewResourceGroup(RG_NAME)
-                    .withNewManageRule("mngRule1")
-                    .withNewSendRule("sndRule1")
+                    .withKafka()
                     .create();
 
+        Assert.assertNotNull(namespace);
+        Assert.assertNotNull(namespace.inner());
+        Assert.assertTrue(namespace.isKafkaEnabled());
+
+        EventHubNamespace namespace2 = eventHubManager.namespaces()
+                .define(namespaceName)
+                .withRegion(REGION)
+                .withNewResourceGroup(RG_NAME)
+                .create();
+
+        Assert.assertNotNull(namespace);
+        Assert.assertNotNull(namespace.inner());
+        Assert.assertFalse(namespace.isKafkaEnabled());
```

Comment:
@thamer-ea 
As of now the tests are failing with this assertion... Kafka Enabled seems to be false